src/main/java/racingCar/RacingCar.java

```diff
@@ -0,0 +1,91 @@
+package racingCar;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RacingCar {
+
+    String inputName;
+    int inputTryNumber;
+
+    ArrayList<Car> garage;
+
```

Comment:
```suggestion

    List<Car> garage;

```

ArrayList 로 사용하는 것 보다 List로 사용하는 것이 조금 더 유연하게 사용할 수 있을 것 같아요 구현체가 변경되어도 이 부분만 변경하면 되기 때문이에요.